dvc/command/get.py

```diff
@@ -31,7 +31,7 @@ def run(self):
 
 
 def add_parser(subparsers, parent_parser):
-    GET_HELP = "Download/copy files or directories from DVC repository."
+    GET_HELP = "Download/copy files or directories from Git repository."
```

Comment:
Maybe let's use
```suggestion
    GET_HELP = "Download a file or directory from any DVC project or Git repository."
```
to match https://dvc.org/doc/command-reference/get